lighthouse-core/runner.js

```diff
@@ -172,13 +172,12 @@ class Runner {
           driverMock: options.driverMock,
         });
 
-        // Calling log.timeEnd here will retroactively update artifacts.Timing.
-        // Must end time here so this timing entry can be stored on saved artifacts.
```

Comment:
The smoke found its first problem 🎉 . This retroactive behavior doesn't happen in DevTools.